client/src/use/useStyling.ts

```diff
@@ -84,6 +84,13 @@ function generateColors(numColors: number) {
 
 export default function useStyling({ markChangesPending }: UseStylingParams) {
   const vuetify = inject('vuetify') as Vuetify;
+
+  /**
+   * Revision counter should be watched for re-rendering based on customStyles
+   * because we're using geojs's styling functions, which will not be invoked until render,
+   * so we can't use deep watching to trigger re-render (chicken and egg).
+   */
+  const customStylesRevisionCounter = ref(1);
```

Comment:
Just curious, why an integer counter and not a flag that's toggled?
I realize that you would have to || the noop conditional if a boolean was used.

Reply:
Consistency, mostly, since counters have been used elsewhere.  I also like that it provides instrumentation if I'm, debugging the event loop.  Easy to see how many times the update was triggered.